clients.json

```diff
@@ -90,6 +90,15 @@
   },
 
   {
+    "name": "go-redis/redis",
+    "language": "Go",
+    "repository": "https://github.com/go-redis/redis"
+    "description": "Redis client for Golang.",
```

Comment:
Probably mention that it supports Redis Sentinel and Redis Cluster out of the box.